src/views/AuthorList.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+export default function AuthorList() {
+  const authors = [
+    { id: 1 },
+    { id: 2 },
+  ];
+
+  return (
+    <div>
+      <Link to="/">Back</Link>
```

Comment:
Pondría "Home" como texto más que "Back". Si la persona entra directamente por la URL de authors, entonces no iría atrás realmente.
```suggestion
      <Link to="/">Home</Link>
```